docs/README.md

```diff
@@ -879,3 +880,15 @@ SET LOCAL que.skip_notify TO true;
 DELETE FROM que_jobs WHERE finished_at < (select now() - interval '7 days');
 COMMIT;
 ```
+
+## Que Jobs Ext View
```

Comment:
Markdown standard is to have a blank line between different elements/paragraphs - so here and after 'Additional Columns:'.

And downcase 'view' to match the headings capitalisation style.